.gitignore

```diff
@@ -38,3 +38,7 @@ pip-log.txt
 coverage.xml
 pip-wheel-metadata
 .test-results/
+
+# mypy
+.mypy_cache
+out
```

Comment:
What is this?

Reply:
both folders are created by mypy.

Reply:
Is it configurable? Can you point it elsewhere?

Reply:
TBH, I did not bother to investigate, I know that the first one is configurable but it needs to be kept inside repository as it cannot work as a shared cache (so basically no benefits from custmizing it). The 2nd one, not sure exactly what/when it appears but I can have a look. 
I can remove it as CI/CD will not complain about it, but probably you will not be very happy to discover it appearing later ;)

Reply:
At least add a comment hinting why it shouldn't be removed from gitignore during refactoring

Reply:
I removed it and I will add it back once I get the same error again, mostly because I want to find out who is really generating this folder.